Gopkg.toml

```diff
@@ -83,6 +83,15 @@ required = [
   branch = "master"
   name = "github.com/wadey/gocovmerge"
 
+
 [[override]]
   name = "github.com/golang/protobuf"
   version = "1.3.1"
+
+[[constraint]]
+  name = "github.com/redhat-developer/devconsole-api"
+  packages = [
+    "pkg/apis",
+    "pkg/apis/devconsole/v1alpha1",
+  ]
```

Comment:
@corinnekrych may I know what was the idea behind adding the `packages` key here? If I'm not mistaken, this is an invalid key for the `[[constraint]]` - packages are allowed only for `[[projects]]` in `Gopkg.lock` file